### PR TITLE
add code js-tiny-world

### PR DESCRIPTION
**Pull request by Oodmincheg**
_Complete the form below and remove this line._

### task name

[Demo](https://oodmincheg.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/oodmincheg/a-tiny-JS-world/blob/master/index.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Oodmincheg/js-tiny-world/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const Inhabition = function(specie, name, gender, legs, hands, saying){
+    this.specie = specie;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying
+    this.hands == 0 ? this.handsString = "no" : this.handsString = this.hands
```
[the remark this answers is not in the document]
Comment (Oodmincheg) [reply submissions/Oodmincheg/js-tiny-world/index.js]: Done.

File: submissions/Oodmincheg/js-tiny-world/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const Inhabition = function(specie, name, gender, legs, hands, saying) {
+  this.specie = specie;
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.saying = saying;
+  hands === 0 ? (this.handsString = "no") : (this.handsString = this.hands);
```
Comment (OleksiyRudenko) [inline submissions/Oodmincheg/js-tiny-world/index.js]: The ternary use here can be generalized as
`condition ? result = whenConditionTrue : result = whenConditionFalse;`
while more common use ternary implementation would follow the pattern
`result = condition ? whenConditionTrue : whenConditionFalse;`

Please, compare the above approaches and fix as appropriate.

Comment (Oodmincheg) [reply submissions/Oodmincheg/js-tiny-world/index.js]: Sorry I've been neglectful.

File: submissions/Oodmincheg/js-tiny-world/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const Inhabition = function(specie, name, gender, legs, hands, saying) {
+  this.specie = specie;
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.saying = saying;
+  this.handsString = hands === 0 ? "no" : this.hands;
```
Comment (OleksiyRudenko) [inline submissions/Oodmincheg/js-tiny-world/index.js]:
```suggestion
  this.handsString = hands === 0 ? "no" : hands;
```
There is no `hands` property (`this.hands`) in this class.

File: submissions/Oodmincheg/js-tiny-world/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  4,
+  0,
+  "MRRRR. PIIICLEEESSSS GOOOOD BOOOOOYYYY"
+);
+const cat = new Inhabition(
+  "cat",
+  "Felix",
+  "male",
+  "4",
+  "0",
```
Comment (OleksiyRudenko) [inline submissions/Oodmincheg/js-tiny-world/index.js]: Your demo says:
```
This world is inhabited by
(world's source code)

"Felix the cat. The wonderful, wonderful cat!"
 Hi. I'm Felix. I'm a cat. I have 0 hands. I have 4 legs. And I'm a male
 
"MRRRR. PIIICLEEESSSS GOOOOD BOOOOOYYYY"
 Hi. I'm Mr. Pickles. I'm a dog. I have no hands. I have 4 legs. And I'm a male
```

Please, note that Felix' zero hands surpass your test `hands === 0`.
You may benefit from a simple method of numeric strings casting to numeric values, e.g. `"0" !== 0` while `+"0" === 0`.

This is definitely a bug. Please, fix. Consider also self-check and debug the entire code before next deployment.

Comment (Oodmincheg) [reply submissions/Oodmincheg/js-tiny-world/index.js]: Sorry one more time :( 

> +"0" === 0
Is there a difference with "0" == 0 ?
Done whith casting.

**Pull request merged**